src/lib/cache/AccessibilityCloudFeatureCache.js

```diff
@@ -101,6 +101,52 @@ export default class AccessibilityCloudFeatureCache extends FeatureCache<
     return uploadPromise;
   }
 
+  ratePlace(
+    placeId: string,
+    mode: 'toilet' | 'wheelchair',
+    rating: 'yes' | 'no' | 'unknown' | 'partial',
+    appToken: string
+  ): Promise<boolean> {
+    const uploadPromise = new Promise((resolve, reject) => {
+      this.constructor
+        .fetch(
+          `${env.REACT_APP_ACCESSIBILITY_APPS_BASE_URL ||
+            ''}/place-infos/rate?id=${placeId}&mode=${mode}&rating=${rating}&appToken=${appToken}`,
+          {
+            method: 'POST',
+            cache: 'no-cache',
+            headers: {
+              Accept: 'application/json',
+              'Content-Type': 'application/json',
+            },
+          }
+        )
+        .then((response: Response) => {
+          if (response.ok) {
+            response
+              .json()
+              .then(json => {
+                resolve(json.success);
+              })
+              .catch(reject);
+          } else if (response.json) {
```

Comment:
Is there a case where response.json is undefined?

Reply:
Yes, if the requests aborts due to cors problems in ff